packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-java-visitor.ts

```diff
@@ -737,13 +739,19 @@ export class AppSyncModelJavaVisitor<
     const annotations: string[] = model.directives.map(directive => {
       switch (directive.name) {
         case 'model':
-          return `ModelConfig(pluralName = "${this.pluralizeModelName(model)}")`;
-          break;
+          const modelArgs: string[] = [];
+          const authDirectives: AuthDirective[] = (model.directives.filter(d => d.name === 'auth') as any) as AuthDirective[];
```

Comment:
Can we use 
```
(model.directives.filter(d => d.name === 'auth') as CodeGenDirective)
```
instead of any

Reply:
Sure, d is already of type CodeGenDirective so we can remove the `as any`.